strawberryfields/apps/qchem/dynamics.py

```diff
@@ -250,3 +254,62 @@ def prob(samples: list, excited_state: list) -> float:
         raise ValueError("The excited state must not contain negative values")
 
     return samples.count(excited_state) / len(samples)
+
+
+def marginal(mu: np.ndarray, V: np.ndarray, n: int, hbar: float = 2.0) -> np.ndarray:
```

Comment:
```suggestion
def marginal(mu: np.ndarray, V: np.ndarray, n_max: int, hbar: float = 2.0) -> np.ndarray:
```

Reply:
I think it's helpful to clarify that this is a maximum number of quanta in the single-mode distribution